sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/IdentitySslUtil.java

```diff
@@ -103,7 +99,7 @@ public void checkServerTrusted(X509Certificate[] certificates, String authentica
 
     private static String extractCertificateThumbprint(Certificate certificate, ClientLogger logger) {
         try {
-            StringBuffer thumbprint = new StringBuffer();
+            StringBuilder thumbprint = new StringBuilder();
```

Comment:
Use StringBuilder instead of StringBuffer which uses locking and this is used in a thread-safe fashion